html/browsers/windows/noreferrer--cross-origin-window-name-manual.sub.html

```diff
@@ -1,3 +1,3 @@
 <meta charset=utf-8>
 <p>Follow this link to open a new browsing context and then confirm it says "idonteven":
-<a rel=noreferrer target=idonteven href="//天気の良い日.{{location[host]}}/html/browsers/windows/support-window-name-echo.html">link</a>.
+<a rel=noreferrer target=idonteven href="//天気の良い日.{{location[host]}}/html/browsers/windows/support-noreferrer--cross-origin-window-name.html">link</a>.
```

Comment:
Hah! @jugglinmike wondered the same thing when he reviewed.

OK, there are two things going on here, one which I'd argue to keep and one which I'm happy to ditch.

* Thing I'd propose keeping: having `support-*` filenames match/pattern to the test file(s) they support. When I started, filenaming was ad hoc in that regard and it was eye-crossing to keep track of which support files went to which test files.
* Thing I'm not attached to: The actual filenaming convention specifics. The double-hyphens you're seeing are inspired by [BEM](http://getbem.com/) CSS component conventions. The thing I was trying to do was delimit the general thing being tested (`noreferrer`, `browsing-context`) from the thing or set of things being tested _on_ it in the given file. However, it sounds like my approach may be slowing people down versus helping, so I'm happy to do away with it.